src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreTests.java

```diff
@@ -1637,6 +1643,13 @@ public void changeSettingsOnRestoreTest() throws Exception {
                 .setIndexSettings(newIncorrectIndexSettings)
                 .setWaitForCompletion(true), SnapshotRestoreException.class);
 
+        logger.info("--> try restoring while changing negative to the number of replicas - should fail");
```

Comment:
I think it would sound better if you changed it to "--> try restoring while changing the number of replicas to a negative number - should fail"